staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -7662,3 +7678,19 @@ const (
 	// the destination set to the node's IP and port or the pod's IP and port.
 	LoadBalancerIPModeProxy LoadBalancerIPMode = "Proxy"
 )
+
+// ImageVolumeSource represents a image volume resource.
+type ImageVolumeSource struct {
+	// Required: Image or artifact reference to be used.
+	// Behaves in the same way as pod.spec.containers[*].image.
+	// More info: https://kubernetes.io/docs/concepts/containers/images
+	Reference string `json:"reference" protobuf:"bytes,1,opt,name=reference"`
```

Comment:
a container's image field is optional to allow injection from higher level construct/controllers when embedded inside of say a deployment.  Do the same forces apply?  if so, update this to optional.  if not, please explain why.

Reply:
Should become optional then, updated the field.

Reply:
I think they could, since the content could be provided by a higher level injection.  Good catch, I missed that.